src/main/java/group1/intern/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,16 @@
+package group1.intern.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+
+@Controller
+public class ShoppingCartController {
+
+    @GetMapping("/your-cart")
```

Comment:
```suggestion
    @GetMapping("/carts")
```
bớt vẽ thêm nha :D dù hay đó nhưng cứ theo chuẩn mà làm, người ta đang giao tiếp tiếng việt mi nói tiếng lào làm j